### PR TITLE
[INJI-692&694] fix overlay height and background border color in purple

### PR DESCRIPTION
**Pull request by srikanth716**
Screenshot_1704714131
Screenshot_1704722982

### CODE REVIEW

File: components/MessageOverlay.tsx [hunk 1]
```diff
@@ -125,5 +121,5 @@ export interface MessageOverlayProps {
   onRetry?: () => void;
   onBackdropPress?: () => void;
   onShow?: () => void;
-  customHeight?: number | string | undefined;
+  minHeight?: number | string | undefined;
```
Comment (KiruthikaJeyashankar) [inline components/MessageOverlay.tsx:124]: We are passing customHeight prop from other components as well, would this prop name change affect there? Can we rename in consumers as well?

Comment (srikanth716) [reply components/MessageOverlay.tsx:124]: Changed on all the other consumers

File: components/MessageOverlay.tsx [hunk 2]
```diff
@@ -10,11 +10,7 @@ export const MessageOverlay: React.FC<MessageOverlayProps> = props => {
   const {t} = useTranslation('common');
   const style = StyleSheet.create({
     customHeight: {
```
Comment (Alka1703) [inline components/MessageOverlay.tsx:12]: I still see occurance of customHeight in ScanLayoutController.ts(line 95) and App.tsx(Line 97). Please have a look.

Comment (srikanth716) [reply components/MessageOverlay.tsx:12]: fixed and updated the pr

**Review by KiruthikaJeyashankar — APPROVED** [review]

**Review by swatigoel — APPROVED** [review]

**Pull request merged**